.env.example

```
@@ -14,8 +14,15 @@ PUBLIC_STELLAR_NATIVE_ASSET_CONTRACT_ID=CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2F
 PUBLIC_STELLAR_SIMULATION_ACCOUNT=GXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 PUBLIC_UNLOCK_PUBLIC_KEY=BASE64_PUBLIC_KEY
 
-# Optional external chat gateway used by the current UI
-PUBLIC_CHAT_API_BASE=https://secret-ai-gateway.onrender.com
+# Optional external chat gateway (relative to backend or full URL)
+PUBLIC_CHAT_API_BASE=
+
+# Backend AI Service Configuration
+AI_PROVIDER_URL=https://secret-ai-gateway.onrender.com
+AI_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AI_API_KEY key should go before the AI_PROVIDER_URL key

(UnorderedKey)